docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToArray.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Conversions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingStringToArrayExample
+    {
+        /// <summary>
+        /// Helper class for retrieving the resulting data
+        /// </summary>
+        class SampleInfertDataWithIntArray
+        {
+            public float Age = 0;
+            public string Education = default;
+            public int[] EducationCategory = default;
+        }
+
+        ///<summary>
+        /// This example demonstrates the use arrays as the values for the ValueMappingEstimator. It maps a set of keys that are type string
+        /// to a integer arrays of variable length.
+        /// The mapping looks like the following:
+        /// <list>
+        ///     <item>0-5yrs -> 1,2,3,4</item>
+        ///     <item>6-11yrs -> 5,6,7</item>
+        ///     <item>12+yrs -> 42, 32</item>
+        /// </list>
+        /// </summary>
+        public static void Run()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Creating a list of keys based on the Education values from the dataset
+            var educationKeys = new List<string>()
+            {
+                "0-5yrs",
+                "6-11yrs",
+                "12+yrs"
+            };
+
+            // Sample list of associated array values
+            var educationValues = new List<int[]>()
+            {
+                new int[] { 1,2,3,4 },
+                new int[] { 5,6,7 },
+                new int[] { 42, 32 }
+            };
+
+            // Constructs the ValueMappingEstimator making the ML.net pipeline
+            var pipeline = new ValueMappingEstimator<string, int>(ml, educationKeys, educationValues, ("EducationCategory", "Education"));
```

Comment:
>r<string, int> [](start = 51, length = 14)

can we merge this sample with the other one? maybe have a pipeline with the two estimators?

can we merge all this samples into one, with a pipeline with more than one estimator?